<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>listaForm</class>
 <widget class="QWidget" name="listaForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lista</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Termino</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tblTemas"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Concepto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tblTerminos"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Asignatura</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="cmbAsignaturas"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
